integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java

```diff
@@ -1,17 +1,25 @@
-package com.sequenceiq.it.cloudbreak.newway;
+package com.sequenceiq.it.cloudbreak.newway.entity.ldap;
 
 import java.util.function.BiConsumer;
 import java.util.function.Function;
 
 import com.sequenceiq.it.IntegrationTestContext;
+import com.sequenceiq.it.cloudbreak.newway.Assertion;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.GherkinTest;
+import com.sequenceiq.it.cloudbreak.newway.Prototype;
+import com.sequenceiq.it.cloudbreak.newway.ResourceAction;
+import com.sequenceiq.it.cloudbreak.newway.TestParameter;
 import com.sequenceiq.it.cloudbreak.newway.action.Action;
+import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigDeleteAction;
 import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigPostAction;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 import com.sequenceiq.it.cloudbreak.newway.v4.LdapConfigV4Action;
 
 @Prototype
 public class LdapConfig extends LdapConfigEntity {
-    private static final String LDAPCONFIG = "LdapCONFIG";
+
+    private static final String LDAPCONFIG = "LDAP_CONFIG";
 
     private LdapConfig() {
```

Comment:
remove old Ldap tests and related codes

Reply:
Can I delete every old tests that use the old IntegrationTestContext and this Ldap?